run_worker.py

```diff
@@ -0,0 +1,26 @@
+# @@@SNIPSTART python-money-transfer-project-template-run-worker
+import asyncio
+
+from temporalio.client import Client
+from temporalio.worker import Worker
+
+from activities import deposit, refund, withdraw
+from shared import MONEY_TRANSFER_TASK_QUEUE_NAME
+from workflows import MoneyTransfer
+
+
+async def main() -> None:
+    client: Client = await Client.connect("localhost:7233", namespace="default")
+    # Run the worker
+    worker: Worker = Worker(
+        client,
+        task_queue=MONEY_TRANSFER_TASK_QUEUE_NAME,
+        workflows=[MoneyTransfer],
+        activities=[withdraw, deposit, refund],
+    )
+    await worker.run()
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
```

Comment:
Probably doesn't matter for this sample, but should be noted this does not perform a graceful worker shutdown (we go back and forth on samples too because it often doesn't matter)